q

```diff
@@ -0,0 +1,88 @@
+[1mdiff --git a/doc/classes/Sprite.xml b/doc/classes/Sprite.xml[m
```

Comment:
That's not meant to be here :)

Reply:
Lol, indeed. And it has conflicts now. ^^